### ISSUE TITLE
Is there anyone working on this projekt?

### ISSUE THREAD

**Issue by shadow73s** [linuxmint/nemo#2366]
Waiting since months for simple Bugfixes like copybar at = wrong%, wrong placed context menu, etc...

Or is the Project dead?

```
 * Nemo version (4.4.2)
 * Is issue with desktop or windowed nemo? Many Bugs
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 19.3
 * 64 bit
 ```

**Comment by smurphos** [comment]
https://github.com/linuxmint/nemo/commits/master

Appears so. ;-)

**Comment by robo2bobo** [comment]
From what I see, most development happened during 2012-2016. After that, real development stopped and its just maintenance and minor patches/updates.

In case a dev is reading this, please fix bug #2291  :)
